templates/default/docker.yaml.erb

```diff
@@ -0,0 +1,50 @@
+# Generated by Chef, local modifications will be overwritten
+
+init_config:
+  <% if @init_config %>
+  <% if @init_config.key?('docker_root') -%>
+  docker_root: <%= @init_config['docker_root'] %>
+  <% end -%>
+  <% if @init_config.key?('socket_timeout') -%>
+  socket_timeout: <%= @init_config['socket_timeout'] %>
+  <% end -%>
+  <% end -%>
+
+instances:
+  <% @instances.each do |i| -%>
```

Comment:
In that case we don't have any default value for url and new_tag_name. Can we suppose it is users responsibility to always define these attributes?